src/main/java/uk/gov/ons/ctp/response/collection/exercise/representation/CollectionExerciseDTO.java

```diff
@@ -17,10 +20,17 @@
 @NoArgsConstructor(access = AccessLevel.PUBLIC)
 public class CollectionExerciseDTO {
 
+  public interface PostValidation {}
+  public interface PutValidation {}
+  public interface PatchValidation {}
+
   private UUID id;
 
+  @NotNull(groups = {PostValidation.class, PutValidation.class})
```

Comment:
What are the benefits to having the groups?

Reply:
I don't think we need the groups as everything is being validated (both PUT and POST) and we aren't using PATCH by the looks of it?